packages/components/src/components/breadcrumb/_breadcrumb.scss

```
@@ -70,6 +74,73 @@
     }
   }
 
+  // Overflow Menu overrides
+  .#{$prefix}--breadcrumb-item .#{$prefix}--overflow-menu {
+    position: relative;
+    width: rem(20px);
+    height: rem(20px);
+
+    &:focus {
+      outline: 1px solid $focus;
+    }
+
+    &:hover {
+      background: transparent;
+    }
+
+    // Used to mimic link underline
+    &::after {
+      position: absolute;
+      bottom: 2px;
+      width: rem(12px);
+      height: 1px;
+      background: $hover-primary-text;
+      opacity: 0;
+      transition: opacity $duration--fast-01 motion(standard, productive);
+      content: '';
+    }
+  }
+
+  .#{$prefix}--breadcrumb-item .#{$prefix}--overflow-menu:hover::after {
+    opacity: 1;
+  }
+
+  .#{$prefix}--breadcrumb-item
+    .#{$prefix}--overflow-menu.#{$prefix}--overflow-menu--open {
+    background: transparent;
+    box-shadow: none;
+  }
+
+  .#{$prefix}--breadcrumb-item .#{$prefix}--overflow-menu__icon {
+    position: relative;
+    margin-top: rem(8px);
+    transform: rotate(90deg);
```

```suggestion
    transform: rotate(90deg) translateX(rem(4px));
```

I think this should lead to consistent alignment across all browsers, replacing `margin-top` with a CSS transform

---

it's kind of trippy using `translateX` to position it vertically, but that's because of the `rotate(90deg)` since we're just importing the overflow menu component directly, which uses the vertical overflow icon. we could also use `renderIcon` with the horizontal overflow icon and then do this maybe

```suggestion
    transform: translateY(rem(4px));
```